lib/pharos/configuration/host.rb

```diff
@@ -15,6 +15,7 @@ class Host < Dry::Struct
       attribute :user, Pharos::Types::Strict::String.default('ubuntu')
       attribute :ssh_key_path, Pharos::Types::Strict::String.default('~/.ssh/id_rsa')
       attribute :container_runtime, Pharos::Types::Strict::String.default('docker')
+      attribute :no_taint_master, Pharos::Types::Strict::Bool
```

Comment:
Suggestions for a better config option?

Reply:
Could it be a role?

Reply:
`role: worker-master`?

Reply:
IMO mixing taints into roles sounds like a bad idea. :)

Couldn't you, at least theoretically, put a taint on any node, not just on master? If so, `taint: true/false` would make sense in my head. The default would depend on the role, for masters it would default to `true`, for workers to `false`.

Reply:
This is specific to the `node-role.kubernetes.io/master:NoSchedule` taint used by kubeadm for kube master nodes.

Alternatively, we could handle `taints` similarly to `labels`, and that way we could also update them.

It seems like the kubeadm markmaster phase should know how to toggle the taint on/off, but `kubeadm upgrade` doesn't actually seem to run the markmaster phase, so currently this flag can only be used for the initial cluster up... unless we explicitly called `kubeadm alpha phase mark-master` instead.